.github/workflows/release.yml

```diff
@@ -22,8 +22,17 @@ jobs:
           registry-url: "https://registry.npmjs.org"
       - name: Build NPM Package
         run: deno task dnt ${{steps.get_tag_version.outputs.TAG_VERSION}}
-      - name: Publish NPM Package
+      - run: npm pack
+        working-directory: "./target/npm"
+      - uses: actions/upload-artifact@v3
+        with:
+          name: package
+          path: "./target/npm/*.tgz"
+      - uses: octokit/request-action@v2.x
         if: startsWith(github.ref, 'refs/tags/')
+        with:
+          route: POST /repos/paritytech/npm_publish_automation/actions/workflows/publish.yml/dispatches
+          ref: main
+          inputs: '${{ format(''{{ "repo": "{0}", "run_id": "{1}" }}'', github.repository, github.run_id) }}'
```

Comment:
It may be easier to use the `gh` cli

```
echo '{...}' | gh workflow run -R paritytech/npm_publish_automation ...
```

Reply:
Perhaps, but this is tested and working, so I'm inclined to leave it as is